src/Couchbase.Lite.Shared/Serialization/MCollection.cs

```diff
@@ -122,7 +122,7 @@ protected void Mutate()
 
         public virtual void Dispose()
         {
-            Context?.Dispose();
+            
```

Comment:
Why make this `IDisposable` just to have a blank method?  You had a better idea before with disposing the context here and then calling base if you needed to extend it (but I'm still not sure that any of these classes should be disposing the context.  They didn't before right?

Reply:
MRoot did dispose Context before. MRoot extends MCollection.

Reply:
What benefit does it add to make `MCollection` an `IDisposable` object?